src/main/kotlin/onm/house/devices/DeviceState.kt

```diff
@@ -0,0 +1,32 @@
+package onm.house.devices
+
+import onm.configuration.DeviceType
+import onm.configuration.PowerConsumption
+
+
+data class DeviceState(val currentPowerConsumption: Int, val isDeviceAvailable: Boolean)
+
+class DeviceStateMachine(
+        private val powerConsumption: PowerConsumption,
+        private val deviceType: DeviceType) {
```

Comment:
Information about power consumption are already included in data class DeviceType. Isn't one of this stat redundant then?

Reply:
Nope, `DeviceType` is enum and has only default values, this will use user defined values